pontoon/base/views.py

```diff
@@ -1105,11 +1105,17 @@ def request_locale(request):
     subject = '[Pontoon] Locale Request'
     message = 'Add locale %s (%s) to project %s (%s).' % (
         locale.name, locale.code, project.name, project.slug)
-    sender = request.user.email
 
     if settings.ADMINS[0][1] != '':
-        recipients = [settings.ADMINS[0][1]]
-        send_mail(subject, message, sender, recipients)
+        sender = 'pontoon@mozilla.com'
+        recipient = settings.ADMINS[0][1]
```

Comment:
While we're here, maybe have a default or raise an appropriate error due to lack of ADMINs? Not a blocker, just a thought.